openwisp_radius/api/freeradius_views.py

```diff
@@ -106,7 +106,9 @@ def _radius_token_authenticate(self, request):
             return self._check_client_ip_and_return(request, cached_orgid)
         else:
             try:
-                radtoken = RadiusToken.objects.get(user__username=username)
+                radtoken = RadiusToken.objects.get(
+                    user=auth_backend.get_users(username).first()
```

Comment:
what if `first()` returns `None`, could you add a test for this case and ensure it's handled well?

Reply:
It will throw does not exist exception if the user is unknown so it was already handled.